spec/unit/check_board_spec.rb

```diff
@@ -15,13 +16,26 @@
                               Player.new(:X, 2, 2)
                             ])
     view_board = ViewBoard.new(player_gateway: player_gateway)
```

Comment:
You've created a [sociable test](https://martinfowler.com/bliki/UnitTest.html) for `CheckBoard` which depends on `ViewBoard`. While this will work, it increases the complexity of your test setup. You can probably make a test double for `ViewBoard` which will enable you to simplify these unit tests.